GliaWidgets/Sources/ViewController/Chat/ChatViewController.swift

```diff
@@ -172,3 +172,22 @@ class ChatViewController: EngagementViewController, MediaUpgradePresenter,
         )
     }
 }
+
+extension ChatViewController {
+    func rebind(viewModel: SecureConversations.ChatWithTranscriptModel) {
+        // Binding is only possible for `ChatView` so casting is required.
+        guard let chatView = view as? ChatView else { return }
+        // Swap transcript model with chat model.
+        self.viewModel = viewModel
+        // Swap existing engagement transcript model
+        // (though it is technically not for engagement)
+        // with engagement chat model in superclass EngagementViewController.
+        super.rebind(engagementModel: viewModel.engagementModel)
```

Comment:
It looks a bit confusing with `rebind` and then `bind`. What does one do that the other doesn't, and is there a way to describe this in the names?